### PR TITLE
#155820624 Connects detail activity to main list

### PR DESCRIPTION
**Pull request by mubstimor**
### What does this PR do?
- Connects the user's layout to main activity with intents

#### Tasks to be completed
- add recycler adapter to main file
- pass intent to detail activity
- add picasso library
- display image from remote url
- display image in a circle

#### Relevant PT stories
[155820624](https://www.pivotaltracker.com/story/show/155820624)

#### Screenshots
- Main layout
Screen Shot 2019-03-27 at 8 08 52 AM

- User Detail layout
Screen Shot 2019-03-27 at 10 11 53 AM

### CODE REVIEW

File: app/src/main/java/mubstimor/android/codelab/CircleTransform.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package mubstimor.android.codelab;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+
+import com.squareup.picasso.Transformation;
+
+/**
```
Comment (joeeasy) [inline app/src/main/java/mubstimor/android/codelab/CircleTransform.java:10]: Please walk me through what you were trying to achieve here

**Review by joeeasy — CHANGES_REQUESTED** [review]
Good job 👍🏻, Kindly attend to the comments

File: app/src/main/java/mubstimor/android/codelab/CircleTransform.java (hunk 1 above)
Comment (mubstimor) [reply app/src/main/java/mubstimor/android/codelab/CircleTransform.java:10]: I was trying to add a transformation to change the image from a default square into a circular one. This script was written as an extra to the picasso library.

Comment (joeeasy) [reply app/src/main/java/mubstimor/android/codelab/CircleTransform.java:10]: Okay, @mubstimor Please remove that comment and add something more descriptive.

Comment (mubstimor) [reply app/src/main/java/mubstimor/android/codelab/CircleTransform.java:10]: @joeeasy I have updated the comment.

**Pull request merged**